documentation/src/main/asciidoc/userguide/appendices/LegacyBasicTypeResolution.adoc

```diff
@@ -0,0 +1,346 @@
+:sourcedir: ../../../../test/java/org/hibernate/userguide/mapping
+:extrasdir: extras
+:originalextrasdir: ../chapters/domain/extras
+
+[[basic-legacy]]
+== Legacy resolution
+
+Versions prior to 6.0 statically combined the `JavaTypeDescriptor`, `JdbcTypeDescriptor`, `BasicValueConverter` and
+`MutabilityPlan` aspects within the `BasicType` contract.  To influence this mapping, users needed to specify the
+'BasicType' or `UserType` (which gets wrapped in a `BasicType`) to use instead
+
+That said, both `BasicType` and `UserType` continue to be fully supported.  See <<basic-legacy>> for details.
+
+Hibernate's legacy strategy for resolving a basic type is based on finding the implementation
+of `org.hibernate.type.BasicType` to use.  Internally, a registry of basic types is used.  The
```

Comment:
"basic type" is the JPA term here.  So no, in this section they are not the same.  `BasicType` simply happens to be one way we understand a "basic type" in how JPA means it.

I do agree though that it is confusing to talk about "type" and "basic type", etc because it gets confusing